README.md

```diff
@@ -4,76 +4,29 @@
 
 #### [Demo](https://bo.wix.com/pages/wix-ui-tpa) | [Demo App](https://github.com/wix/wix-ui-tpa-example)
   
-## Installation
+## 📦 Installation
 * Install with `npm` or `yarn`:
 ```sh
 npm i wix-ui-tpa
 # OR
 yarn add wix-ui-tpa
 ```
  
-## Prerequisites
+## 🖇 Prerequisites
 `wix-ui-tpa` uses [Stylable](https://stylable.io/) configuration, and depends on 
 [`wix-style-processor`](https://github.com/wix/wix-style-processor) by default.     
 In order to use `wix-ui-tpa`, your module bundler and app should be configured accordingly.
 
-## Usage
-Using components is pretty much like any other React component:
-```typescript jsx
-import React from 'react';
-import { TPAComponentsProvider } from 'wix-ui-tpa/TPAComponentsConfig';
-import { Text, TYPOGRAPHY } from 'wix-ui-tpa/Text';
-import { Button } from 'wix-ui-tpa/Button';
+## 🚀 Usage
+Please refer to the [Usage document](https://github.com/wix/wix-ui-tpa/blob/master/docs/USAGE.md).
```

Comment:
I think if we can refer relatively it would be better:
```suggestion
Please refer to the [Usage document](docs/USAGE.md).
```

I tried it in my recent PR but not sure it actually works (if not we'll need to fix that anyway).

Reply:
The 'shortcut' works when you clicking on the link for the GitHub page but when clicking on it from the storybook it doesn't. This is why I changed it for the full URL.

Reply:
BTW I changed also the 'Contributing' and the 'Release Process' links.

Reply:
OK, so another task for the backlog is adding support of resolving relative links in Storybook. :)